control-plane/subcommand/server-acl-init/command_test.go

```diff
@@ -2544,18 +2546,20 @@ func TestRun_ValidateLoginToken_SecondaryDatacenter(t *testing.T) {
 			})
 			require.NoError(t, err)
 
-			tok, _, err := client.ACL().Login(&api.ACLLoginParams{
-				AuthMethod:  authMethodName,
-				BearerToken: jwtToken,
-				Meta:        map[string]string{},
-			}, &api.WriteOptions{})
-			require.NoError(t, err)
-
-			require.Equal(t, len(tok.Roles), len(c.Roles))
-			for _, role := range tok.Roles {
-				require.Contains(t, c.Roles, role.Name)
-			}
-			require.Equal(t, !c.GlobalToken, tok.Local)
+			retry.Run(t, func(r *retry.R) {
```

Comment:
Can we make this exact change to the corresponding test in `command_ent_test` as well, please?